deploy/storage/base_bigtable_info.yaml

```diff
@@ -2,13 +2,13 @@ project: datcom-store
 instance: prophet-cache
 tables:
   - auto2w_2023_10_03_10_36_05
-  - auto1d_2023_10_11_02_22_32
+  - auto1d_2023_10_12_02_26_14
   - auto1w_2023_10_08_21_16_19
-  - frequent_2023_10_11_05_13_48
+  - frequent_2023_10_12_05_16_10
```

Comment:
can you pls use frequent_2023_10_13_02_46_29 or later that has the 2601A S table reverted:
https://autopush.datacommons.org/tools/timeline#statsVar=Count_Person_PovertyStatusDetermined&place=geoId/06

Reply:
Done. Thanks!